mbof/views.py

```diff
@@ -39,6 +39,25 @@ def get_queryset(self):
         return Event.objects.filter(startTime__lte=current_time, endTime__gte=current_time).order_by('postingTime')
 
 
+class CurrentEventViewSet(EventViewSet):
+    """
+    API endpoint that allows users to be viewed or edited.
```

Comment:
You need to update the docstrings for these classes.  They are describing some other class.